ci/infra/jobs/build_amis/dw-al2-hardened-ami.yml

```diff
@@ -0,0 +1,165 @@
+      - .: (( inject meta.plan.build-ami ))
+        config:
+          params:
+            AWS_ROLE_ARN: arn:aws:iam::((aws_account.management)):role/ci
+          inputs:
+            - name: dw-al2-hardened-ami-pr
+        on_failure:
+          put: dw-al2-hardened-ami-pr
+          params:
+            path: dw-al2-hardened-ami-pr
+            status: failure
+        on_success:
+          put: dw-al2-hardened-ami-pr
+          params:
+            path: dw-al2-hardened-ami-pr
+            status: success
+
+  - name: dw-al2-test-and-promote-hardened-ami
+    max_in_flight: 1
+    plan:
+      - get: dw-al2-untested-hardened-ami
+        trigger: true
+      - get: aws-management-infrastructure
+        trigger: false
+      - get: aws-security-tools
+        trigger: false
+      - .: (( inject meta.plan.terraform-output-management ))
+        config:
+          params:
+            TF_WORKSPACE: management
+      - .: (( inject meta.plan.start-instance ))
+        input_mapping:
+          source_ami: dw-al2-untested-hardened-ami
+        config:
+          params:
+            LAUNCH_TEMPLATE_OUTPUT_NAME: "hardened_test_launch_template"
+      - .: (( inject meta.plan.run-inspector ))
+      - .: (( inject meta.plan.evaluate-assessment-run))
+        config:
+          params:
+            WHITELIST_NAME: whitelist_al2_hardened_ami
+      - .: (( inject meta.plan.promote-ami ))
+        input_mapping:
+          source_ami: dw-al2-untested-hardened-ami
+        config:
+          params:
+            AMI_NAME_PREFIX: "dw-al2-hardened-ami-"
+    ensure:
+      do:
+        - .: (( inject meta.plan.inspector-cleanup ))
+
+  - name: dw-al2-weekly-test-hardened-ami
+    max_in_flight: 1
+    plan:
+      - get: weekly-wednesday
+        trigger: true
+      - get: dw-al2-untested-hardened-ami
+        trigger: true
+      - get: aws-management-infrastructure
+        trigger: false
+      - get: aws-security-tools
+        trigger: false
+      - .: (( inject meta.plan.terraform-output-management ))
+        config:
+          params:
+            TF_WORKSPACE: management
+      - .: (( inject meta.plan.start-instance ))
+        input_mapping:
+          source_ami: dw-al2-untested-hardened-ami
+        config:
+          params:
+            LAUNCH_TEMPLATE_OUTPUT_NAME: "hardened_test_launch_template"
+      - .: (( inject meta.plan.run-inspector ))
+      - .: (( inject meta.plan.evaluate-assessment-run))
+        config:
+          params:
+            WHITELIST_NAME: whitelist_al2_hardened_ami
+    ensure:
+      do:
+        - .: (( inject meta.plan.inspector-cleanup ))
+    on_failure:
+      put: slack
+      attempts: 3
+      params:
+        channel: "#dataworks-alerts"
+        icon_emoji: ":concourse-red:"
+        attachments:
+          - color: danger
+            text: ":fire: Weekly AWS Inspector test of dw-al2-hardened-ami has failed - non-whitelisted vulnerabilities found "
+            attachment_type: "default"
```

Comment:
you have most of this in `meta.yml` but you aren't using it, can you use the one in meta and just change the attachment?

Reply:
Kinda.  You can have multiple attachments, so I dropped the one that wouldn't work anyway. It means its reusable, and you need to just specify the attachment when you inject it.